Environment_Engine/Compute/BooleanIntersect.cs

```diff
@@ -27,21 +27,24 @@
 
 using BH.oM.Environment.Elements;
 using BH.Engine.Geometry;
+using BH.oM.Reflection.Attributes;
+using System.ComponentModel;
 
 namespace BH.Engine.Environment
 {
     public static partial class Compute
     {
         /***************************************************/
-        /****          public Methods - Lines           ****/
+        /****          public Methods                   ****/
         /***************************************************/
 
-        public static bool DoesBooleanIntersect(this BuildingElement element, BuildingElement elementToCompare)
+        [Description("BH.Engine.Environment.Compute.BooleanIntersect => Returns true if two Environment Panels boolean intersect")]
```

Comment:
Do not agree with typing out the name of the method at the start of each description. If this is something we want in general, it should be handled in the reflection_Engine or BHoM_UI, as the name and namespace fo the method can be accessed with ease. This to me just add more maintenance and risk of spelling errors etc. Might also be made somewhat redundant with the "Source Code" option added by @epignatelli in GH if we can get that added to dynamo and excel as well.